packages/angular-workspace/nimble-angular/README.md

```diff
@@ -143,7 +143,7 @@ To avoid this, call `processUpdates()` after each `fakeAsync` test. This will sy
 
```

Comment:
Maybe we need some docs in Contributing to make sure to align to supported angular associated libraries (typescript, rxjs) when bumping angular versions